tests/wicked/before_test.pm

```diff
@@ -31,7 +32,9 @@ sub run {
     my $escaped                = $enable_command_logging =~ s/'/'"'"'/gr;
     assert_script_run("echo '$escaped' >> /root/.bashrc");
     assert_script_run($enable_command_logging);
-    unless (is_sle('15+')) {    # image which we using for sle15 don't have firewall running
+    # image which we using for sle15 don't have firewall running.
+    # QAM need another way to figure out firewall state due to wider set of images
+    if (is_sle('<15') || (is_updates_tests() && systemctl("is-active" . opensusebasetest::firewall))) {
```

Comment:
maybe 1 condition would be enough?  `if systemctl("is-active" . opensusebasetest::firewall))`

then, we stop/disable firewall in all cases where it's active... regardless of SLE version

Reply:
I wanted to avoid this , because in case of SLE 15 it is obvious that we don't have it , so it is pure time wasting type in console 'systemctl is-active firewalld'

Reply:
fair enough.

Reply:
missing space in string concatenation: https://openqa.suse.de/tests/3466413#step/before_test/76

Reply:
@Hoorhay  #8653 **suppose** to fix it , but I want you to check it **before** saying it is LGTM

Reply:
I was not able to run wicked tests on my instance, even after following documents for multi machine setup. So there's no way for me to check anyway.

Reply:
Let me RTFM you:
https://github.com/os-autoinst/openQA/blob/master/docs/WritingTests.asciidoc#triggering-tests-based-on-an-any-remote-git-refspec-or-open-github-pull-request

Reply:
if you plan to run wicked tests in regular basis you **need** figure out how to verify patches . Because in long term I won't provide support for QAM in regular basis. Sorry I have my own backlog which does not contain QAM

Reply:
if you contact me using any way which suitable for you with some certain problem I would be glad to help, but I can't say much about "unable to run wicked tests". Also have you tried to follow suggestion from @coolo  to simply test patch on osd instead ?

Reply:
@Hoorhay take into account that wicked tests need OVS setup in your machine. By following this guide you will be able to have a proper setup: http://open.qa/docs/#_multi_machine_tests_setup 

Basically, those MultiMachine tests are created on bridges/TAP devices on OVS, and not on standard qemu created networks. 

Let us know on IRC/RC if you find any issue setting it up and we'll try to help you.